mindocr/optim/adan.py

```diff
@@ -73,67 +54,55 @@ def _update_run_op(
+    next_v = op_mul(ops.tuple_to_array((1.0,)) - beta2, v_fp32) + op_mul(beta2, gradient_fp32 - prev_gradient_fp32)
 
-    next_n = op_mul(F.tuple_to_array((1.0,)) - beta3, n_fp32) + op_mul(
-        beta3, op_square(gradient + op_mul(F.tuple_to_array((1.0,)) - beta2, gradient_fp32 - prev_gradient_fp32))
+    next_n = op_mul(ops.tuple_to_array((1.0,)) - beta3, n_fp32) + op_mul(
+        beta3, op_square(gradient + op_mul(ops.tuple_to_array((1.0,)) - beta2, gradient_fp32 - prev_gradient_fp32))
     )
 
     lr_t = lr / (eps + op_sqrt(next_n))
 
-    update = next_m + op_mul(F.tuple_to_array((1.0,)) - beta2, next_v)
-
-    # if decay_flag:
-    #    update = op_mul(weight_decay, param_fp32) + update
+    update = next_m + op_mul(ops.tuple_to_array((1.0,)) - beta2, next_v)
 
     next_param = param_fp32 - op_reshape(op_mul(lr_t, update), op_shape(param_fp32))
 
     next_param = next_param / (Tensor(1.0, mstype.float32) + op_mul(weight_decay, lr_t))
 
-    success = F.depend(success, F.assign(param, op_cast(next_param, F.dtype(param))))
-    success = F.depend(success, F.assign(m, op_cast(next_m, F.dtype(m))))
-    success = F.depend(success, F.assign(v, op_cast(next_v, F.dtype(v))))
-    success = F.depend(success, F.assign(n, op_cast(next_n, F.dtype(n))))
-    success = F.depend(success, F.assign(prev_gradient, gradient))
+    success = ops.depend(success, ops.assign(param, op_cast(next_param, ops.dtype(param))))
+    success = ops.depend(success, ops.assign(m, op_cast(next_m, ops.dtype(m))))
+    success = ops.depend(success, ops.assign(v, op_cast(next_v, ops.dtype(v))))
+    success = ops.depend(success, ops.assign(n, op_cast(next_n, ops.dtype(n))))
+    success = ops.depend(success, ops.assign(prev_gradient, gradient))
 
-    return op_cast(next_param, F.dtype(param))
+    return op_cast(next_param, ops.dtype(param))
 
 
-def _check_param_value(beta1, beta2, eps, use_locking, prim_name):
+def _check_param_value(beta1, beta2, eps, prim_name):
     """Check the type of inputs."""
-    assert isinstance(beta1, float), f"For '{prim_name}', the type of 'beta1' must be 'float', but got type '{type(beta1).__name__}'."
-    assert isinstance(beta2, float), f"For '{prim_name}', the type of 'beta2' must be 'float', but got type '{type(beta2).__name__}'."
-    assert isinstance(eps, float), f"For '{prim_name}', the type of 'eps' must be 'float', but got type '{type(eps).__name__}'."
-    assert 0.0 < beta1 < 1.0, f"For '{prim_name}', the range of 'beta1' must be (0.0, 1.0), but got {beta1}."
-    assert 0.0 < beta2 < 1.0, f"For '{prim_name}', the range of 'beta2' must be (0.0, 1.0), but got {beta2}."
-    assert eps > 0, f"For '{prim_name}', the 'eps' must be positive, but got {eps}."
-    assert isinstance(use_locking, bool), f"For '{prim_name}', the type of 'use_locking' must be 'bool', but got type '{type(use_locking).__name__}'."
+    assert isinstance(beta1, float) and 0 <= beta1 <= 1.0, f"For {prim_name}, beta1 should between 0 and 1"
```

Comment:
Update the docstring of function `_update_run_op()` in line 44 and 45. `(0.0, 1.0)` -> `[0.0, 1.0]`